setup.py

```diff
@@ -504,8 +505,21 @@ def build_extensions(self):
                 for d in os.environ[k].split(os.path.pathsep):
                     _add_directory(library_dirs, d)
 
-        _add_directory(library_dirs, os.path.join(sys.prefix, "lib"))
-        _add_directory(include_dirs, os.path.join(sys.prefix, "include"))
+        prefix_lib = os.path.join(sys.prefix, "lib")
+        prefix_include = os.path.join(sys.prefix, "include")
+
+        if sys.prefix != sys.base_prefix:
+            _add_directory(library_dirs, prefix_lib)
+            _add_directory(include_dirs, prefix_include)
+        else:
+            _add_directory(
+                library_dirs,
+                sysconfig.get_config_var("LIBDIR") or prefix_lib,
+            )
+            _add_directory(
+                include_dirs,
+                sysconfig.get_config_var("INCLUDEDIR") or prefix_include,
+            )
```

Comment:
```suggestion
        _add_directory(
            library_dirs,
            (sys.prefix == sys.base_prefix and sysconfig.get_config_var("LIBDIR"))
            or os.path.join(sys.prefix, "lib"),
        )
        _add_directory(
            include_dirs,
            (sys.prefix == sys.base_prefix and sysconfig.get_config_var("INCLUDEDIR"))
            or os.path.join(sys.prefix, "lib"),
        )
```
Maybe this is simpler?

Reply:
I like it, thanks!